changelog.d/19686_element_size_histogram.enhancement.md

```diff
@@ -0,0 +1,2 @@
+Added a new histogram-type metric measuring the byte-size of individual elements received by endpoint sources.
```

Comment:
```suggestion
Added a new histogram metric, `component_received_bytes`, that measures the byte-size of individual events received by the following sources:

- source 1
- source 2
- source 3
```

We should call out each affected source here explicitly to make it easy for readers to understand which sources have this new metric.